### PR TITLE
SC-6086: a11y teams

### PR DESCRIPTION
**Pull request by haniapalac**
# Description

https://ticketsystem.schul-cloud.org/browse/SC-6086

## Approval for review
- [x] All points were discussed with the ticket creator, support-team or product owner. The code upholds all quality guidelines from the PR-template.

> Notice: Please remove the WIP label if the PR is ready to review, otherwise nobody will review it.

### Link to Definition of Done
More and detailed information on the *definition of done* can be found [on Confluence](https://docs.schul-cloud.org/pages/viewpage.action?pageId=92831762)

### CODE REVIEW

File: views/lib/components/table.hbs [hunk 1]
```diff
@@ -32,21 +33,23 @@
                                     {{#each this}}
                                         {{#if this.method}}
                                             <form action="{{this.link}}" method="post" class="inline">
-                                                <input type="hidden" name="_method" value="{{this.method}}"/>
+                                                <label for="method" hidden>{{this.label}}</label>
+                                                <input id="method" type="hidden" name="_method" value="{{this.method}}" hidden />
                                                 {{> "lib/components/csrfInput"}}
-                                                <button type="submit" class="btn btn-secondary btn-sm no-inline-styling {{this.class}}" title="{{this.title}}">
+                                                <button type="submit" class="btn btn-secondary btn-sm no-inline-styling {{this.class}}" title="{{this.title}}" aria-label="{{this.title}}" tabindex="0">
```
Comment (luizzaly) [inline views/lib/components/table.hbs:39]: do we need the title when we have the aria? :)

File: views/teams/topics.hbs [hunk 2]
```diff
@@ -136,16 +136,16 @@
                     data-loclink="/teams/{{../_id}}/tools/show/{{this._id}}">
                     <p class="card-title-directory">
                         {{#if logo_url}}
-                        <img src={{this.logo_url}} class="float-left" />
+                        <img src={{this.logo_url}} class="float-left" alt="logo" />
```
Comment (luizzaly) [inline views/teams/topics.hbs:139]: can we use i18n here please ? :)

File: views/teams/topics.hbs [hunk 3]
```diff
@@ -136,16 +136,16 @@
...
                         {{else}}
-                        <img src="/images/cloud.png" class="float-left" />
+                        <img src="/images/cloud.png" class="float-left" alt="logo" />
```
Comment (luizzaly) [inline views/teams/topics.hbs:141]: same here

**Review by luizzaly — COMMENTED** [review]
nice work! :)

File: views/lib/components/table.hbs (hunk 1 above)
Comment (haniapalac) [reply views/lib/components/table.hbs:39]: It depends on whether we need a tooltip. If not, then aria-label should be sufficient.

**Review by qnowak — APPROVED** [review]

**Review by luizzaly — APPROVED** [review]

**Pull request merged**